### PR TITLE
allow to multi files

### PR DESCRIPTION
**Pull request by sotayamashita**
Firstly. I rarely write ruby so please pardon me 😢 

It's just a suggestion to test multi files. Usually awesome project have one README file so we do not need this feature but awesome bot is good for people to write docs in GitHub. What do you think ?

### CODE REVIEW

**Comment by dkhamsing** [comment]
Thanks for your suggestion!

Will have to review this more later

**Comment by dkhamsing** [comment]
Can you see how it broke the allow options

**Comment by sotayamashita** [comment]
I tried it on my repository. Here is the travis result below:

https://travis-ci.org/sotayamashita/frontend-docs

File: lib/awesome_bot/cli.rb [hunk 1]
```diff
@@ -20,14 +20,27 @@ def make_option(o)
       "--#{o}"
     end
 
+    def divide(argv)
```
Comment (sotayamashita) [inline lib/awesome_bot/cli.rb]: @dkhamsing Here is the code to divide files and params but I know that it is not code.

**Comment by sotayamashita** [comment]
What about using Thor: https://github.com/erikhuda/thor

**Comment by dkhamsing** [comment]
Was thinking of using https://github.com/docopt/docopt.rb 
There are many other projects .. Not sure which to try

**Comment by sotayamashita** [comment]
@dkhamsing so What do you think that it allow files to check for the links ?

**Comment by dkhamsing** [comment]
Can you rephrase your question

**Comment by dkhamsing** [comment]
@sotayamashita can you rephrase your last question

**Comment by dkhamsing** [comment]
Good idea, have to find out what broke CI

**Comment by sotayamashita** [comment]
@dkhamsing 

Sorry for your deleted message not to replay. It is just suggestion so If you agree with this idea, I will reimplement it.

> Good idea, have to find out what broke CI

I got it but, before fixing the problem. I think we should consider whether we use CLI tools or not. What do you think ?

**Comment by dkhamsing** [comment]
Yes, let's first pick a CLI lib

**Comment by dkhamsing** [comment]
@sotayamashita 
Please join in the discussion in #79

Thanks

**Comment by dkhamsing** [comment]
I'll be implementing this as well
Thanks

https://github.com/dkhamsing/awesome_bot/tree/multi-files
